fleetscorebot/Dockerfile.template

```diff
@@ -1,14 +1,14 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-node:9-slim
+FROM resin/%%RESIN_MACHINE_NAME%%-node:10-slim
 
 WORKDIR /usr/src/app
 ENV INITSYSTEM=on
 
 RUN    apt-get update \
-    && apt-get install xz-utils \
+    && apt-get install git xz-utils \
     && apt-get clean
 
 COPY package*.json ./
-RUN npm install
+RUN npm install --only=production && npm cache verify
```

Comment:
You can also remove with `npm cache clean -f`

Reply:
@CameronDiver npm cache verify was trying to hunt down some issues where on my local dev some modules were there, but when deployed on the device, they weren't. Actually that verificationn part is not needed anymore since it was tracked back to the issue of using node 10 locally, and node 9 in deploy...

Reply:
Sure, makes sense.